Tests/RxCocoaTests/UIScrollView+RxTests.swift

```diff
@@ -170,6 +170,53 @@ extension UIScrollViewTests {
         XCTAssertTrue(didScrollToTop)
         subscription.dispose()
     }
+
+    func testDidEndScrollingAnimation() {
+        // direct call `scrollViewDidEndScrollingAnimation`
+        autoreleasepool {
+            let scrollView = UIScrollView()
+            var didEndScrollingAnimation = false
+            
+            let subscription = scrollView.rx.didEndScrollingAnimation.subscribe(onNext: {
+                didEndScrollingAnimation = true
+            })
+            
+            XCTAssertFalse(didEndScrollingAnimation)
+            
+            scrollView.delegate!.scrollViewDidEndScrollingAnimation!(scrollView)
+            
+            XCTAssertTrue(didEndScrollingAnimation)
+            subscription.dispose()
+        }
+        
+        // change contentOffset with animation
+        autoreleasepool {
```

Comment:
Hi, I would be happy to merge this if you removed this autorelease pool closure part of code.

We should be only testing that when delegate fires new element is produced.

Part inside this `autorelease` closure tests correctness of UIScrollView internal logic which is out of scope of this project.